package.json

```diff
@@ -18,6 +18,14 @@
     "nunjucks": "^3.2.3"
   },
   "devDependencies": {
+    "@types/cookie-parser": "^1.4.2",
+    "@types/debug": "^4.1.5",
+    "@types/express": "^4.17.11",
+    "@types/http-errors": "^1.8.0",
+    "@types/morgan": "^1.9.2",
+    "@types/nunjucks": "^3.1.4",
+    "@typescript-eslint/eslint-plugin": "^4.21.0",
```

Comment:
the pr check is complaining it cant find `@typescript-eslint/parser` i guess add it here?

Reply:
Will add